src/pages/Home.jsx

```diff
@@ -2,19 +2,15 @@ import React from 'react'
 
 function Home () {
   return (
-    <div className="pt-16 md:py-8">
-      <div className="pl-24">
-      <div className="pr-24">
-      <h1 className="text-3xl sm:text-4xl text-center">Yorkshire Woods Ground Crew</h1>
-
-      <p className="text-base mt-2.5 sm:mt-7">
-      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Parturient montes nascetur ridiculus mus mauris vitae ultricies leo. Praesent semper feugiat nibh sed pulvinar proin gravida hendrerit lectus. Augue mauris augue neque gravida in fermentum et sollicitudin. Lacus sed viverra tellus in hac habitasse platea. Molestie a iaculis at erat pellentesque adipiscing. Eget nunc scelerisque viverra mauris in aliquam sem fringilla ut. Hac habitasse platea dictumst vestibulum rhoncus. Cras pulvinar mattis nunc sed blandit libero volutpat. Nibh nisl condimentum id venenatis a condimentum vitae sapien pellentesque. Lorem ipsum dolor sit amet consectetur adipiscing elit. Nisl purus in mollis nunc sed id semper. Egestas sed tempus urna et. Elit ut aliquam purus sit amet luctus venenatis lectus magna.
-        <br></br>
-        <br></br>
-      Egestas dui id ornare arcu odio ut sem nulla pharetra. Dictumst quisque sagittis purus sit amet volutpat consequat. Lacus laoreet non curabitur gravida. Semper auctor neque vitae tempus quam pellentesque nec nam. Cras fermentum odio eu feugiat pretium nibh ipsum consequat. Eget felis eget nunc lobortis. Ornare massa eget egestas purus viverra accumsan in nisl. Justo eget magna fermentum iaculis. Amet justo donec enim diam. Sit amet dictum sit amet justo donec enim diam vulputate. Nec tincidunt praesent semper feugiat. Tellus rutrum tellus pellentesque eu tincidunt tortor aliquam. Velit euismod in pellentesque massa. At erat pellentesque adipiscing commodo elit at imperdiet dui accumsan. Odio morbi quis commodo odio aenean sed.
-      </p>
-    </div>
-    </div>
+    <div className="mt-[86px] mx-[34px] md:mt-[160px] md:mx-[364px] bg-white bg-opacity-60 rounded-lg" role="region" aria-label="Business Information">
+      <div className="pl-[32px] pr-[32px] flex flex-col justify-center items-center py-[16px]">
+        <h1 className="text-[50px] md:text-[64px] text-center font-header font-bold" role="heading">Yorkshire Woods Ground Crew</h1>
+        <p className="text-[24px] px-[38px] font-body" role="description">
```

Comment:
```suggestion
        <p className="text-[24px] px-[38px] font-body" role="description">
```
issue: We don't need to provide a role here, since we are not using the `p` tag in any special way. Also, `description` isn't a valid role that I know of. I'd remove the `role="description`.

suggestion: It could be helpful to read through this [guide to aria roles](https://www.a11yproject.com/posts/an-indepth-guide-to-aria-roles/), since we may want to use roles later in the project.